src/migrations/201602231400_initial.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+function createAccountsTable (knex) {
```

Comment:
As discussed with @clark800 , I would recommend some schema changes, such as not using enum, preferring varchar to text, normalizing what's in JSON column.  But we agreed that we'd do it in another ticket, which I will file.